cg/constants/nanopore_files.py

```diff
@@ -0,0 +1,22 @@
+"""Constants related to Oxford Nanopore sequencing."""
+from enum import StrEnum
+from pathlib import Path
+
+
+class NanoporeDirsAndFiles(StrEnum):
+    """Holds file and directory names for nanopore sequencing output."""
+
+    data_directory: str = "nanopore"
+    sequencing_summary_pattern: str = r"final_summary_*.txt"
+
+    @classmethod
+    def sequencing_summary_file(cls, flow_cell_directory: Path) -> Path | None:
+        try:
+            file = Path(next(flow_cell_directory.glob(cls.sequencing_summary_pattern.value)))
```

Comment:
I think we have a utils function for this as well
```suggestion
            file = get_files_matching_pattern(directory: Path, pattern: str)

Reply:
Also does not allow for wildcards in `directory`